python/ray/_private/runtime_env/context.py

```diff
@@ -53,6 +53,7 @@ def exec_worker(self, passthrough_args: List[str], language: Language):
 
         exec_command = " ".join([f"{executable}"] + passthrough_args)
         command_str = " && ".join(self.command_prefix + [exec_command])
+        logger.info(f"Exec'ing worker with command: {command_str}")
```

Comment:
moving the logging to _before_ the command so that the message will be emitted even if the command fails.

Reply:
Maybe this should be in a separate PR if it is causing unsolvable problems.